web/src/components/njwds-layout/SidebarPageLayout.tsx

```diff
@@ -53,14 +53,15 @@ export const SidebarPageLayout = ({
               </div>
               {belowBoxComponent}
             </div>
-            <div className="desktop:grid-col-3 order-first">
-              {isLargeScreen && (
+
+            {isLargeScreen && (
+              <div className="fs0 fg1 order-first">
```

Comment:
[dust] Do we need this `div`? Could these classes just be applied directly to the `nav` element?